test/format/t.c

```diff
@@ -381,12 +381,6 @@ main(int argc, char *argv[])
 skip_operations:
     locks_destroy(g.wts_conn);
 
-    /*
-     * Verify the objects. Verify closes the underlying handle and discards the statistics, read
-     * them first.
-     */
-    TIMED_MAJOR_OP(wts_verify(g.wts_conn, true));
```

Comment:
I think it isn't quite that simple. This verify call we want to keep on a normal run. It verifies after everything completes. So I think it should be:
/* For verify_only we've already completed the verification above. Skip this one. */
if (!verify_only)
    TIMED_MAJOR_OP(wts_verify(g.wts_conn, true));

Reply:
Thanks @sueloverso, I misunderstood the issue.
Is the comment still correct btw:
```
    /*
     * Verify the objects. Verify closes the underlying handle and discards the statistics, read
     * them first.
     */
```
Especially the part that says `read them first`?